JeffShepherd/Rancid-Tomatillos-V2
# Refactor/move api and utilites files

## What does this PR do?

- Moves api.js and utilities.js files from the components folder to the parent src folder
- All relevant import file paths updated to reflect the new locations of these files

## How should it be tested in the UI?

- Run 'npm test' and ensure all tests still pass
- Run 'npm start' and regression test all paths within the application, checking for visual and console log errors
